test/controlplane/node/ciliumnodes/v1.23/state1.yaml

```diff
@@ -88,26 +88,26 @@ items:
       sizeBytes: 459801943
     - names:
       - docker.io/library/import-2022-09-02@sha256:f27fe306ea3c0e3ef176dda243bcbcc2bd8fa0f9d19f87285b0fb47e96d85f93
-      - k8s.gcr.io/kube-proxy:v1.23.10
+      - registry.k8s.io/kube-proxy:v1.23.10
```

Comment:
These changes here are generated from script, and mainly depends on kind + workder node versions, I am not sure if we should update these files, as they can be reverted back next time we run the script.

Reply:
@sayboras Can you link the script that generate those files ? I'm not familiar with the codebase.

Reply:
sure, here is the one for this test

https://github.com/cilium/cilium/blob/master/test/controlplane/node/ciliumnodes/generate.sh

Reply:
You might be able to try this out with `make -C test/controlplane/ generate-input-files`, but it seems that it is sensitive to some local environment settings.